src/ops/sequences.jl

```diff
@@ -15,7 +15,7 @@ end
 
 convert_eltype(x, dtype) = x
 
-@op function constant(value; dtype=nothing, kwargs...)
+@op function constant(value; dtype = nothing, kwargs...)
```

Comment:
I wonder if we shouldn't have this return its own `Constant` type,
and have that type just type-promote into either a `Tensor` or a `TensorHandle` depending on what it touchs?

Reply:
That would make sense, esp if we eliminate having a global eager mode. I think we can merge this for now and then worry about that, though.